activestorage/test/service/mirror_service_test.rb

```diff
@@ -56,7 +56,7 @@ class ActiveStorage::Service::MirrorServiceTest < ActiveSupport::TestCase
   end
 
   private
-    def upload(data, to:)
+    def upload(data)
       SecureRandom.base58(24).tap do |key|
         io = StringIO.new(data).tap(&:read)
         @service.upload key, io, checksum: Digest::MD5.base64digest(data)
```

Comment:
Looks like that `@service.upload` should be `to.upload`.

Reply:
Oh, you got it! Let me fix that.

Reply:
I fixed the test and I think this is the test originally intended!